src/Offer/Events/AbstractOwnerChanged.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\Events;
+
+use Broadway\Serializer\SerializableInterface;
+
+abstract class AbstractOwnerChanged implements SerializableInterface
+{
+    /**
+     * @var string
+     */
+    private $offerId;
+
+    /**
+     * @var string
+     */
+    private $newCreator;
```

Comment:
I think the creator becomes the owner? Maybe better to keep the naming consistent and pick either creator or owner?

Reply:
Yep, it's a leftover from when I called it `CreatorUpdated` first (didn't commit that because I felt that was a weird name) 👍